robot-core/pom.xml

```
@@ -288,6 +288,22 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+    <groupId>org.apache.jena</groupId>
+    <artifactId>apache-jena-libs</artifactId>
+    <version>4.9.0</version>
```

```suggestion
    <artifactId>apache-jena-libs</artifactId>
    <type>pom</type>
    <version>4.9.0</version>
```

This module is of type "POM".